deploy/deadman/install.sh

```
@@ -0,0 +1,103 @@
+paths={
+ "ffe_deadman_watchdog.py":"/usr/local/libexec/ffe-deadman-watchdog",
+ "ffe-deadman.timer":"/etc/systemd/system/ffe-deadman.timer",
+ "README.md":"/usr/local/share/doc/ffe-deadman/README.md",
+}
+for name,digest in m["artifacts"].items():
+    if name in paths: print(f"{digest}  {paths[name]}")
+PY
+)
+grep -Fqx "User=$SERVICE_USER" /etc/systemd/system/ffe-deadman.service
+grep -Fqx "Group=$SERVICE_GROUP" /etc/systemd/system/ffe-deadman.service
+! grep -q '__FFE_DEADMAN_' /etc/systemd/system/ffe-deadman.service
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)